src/main/core/main.c

```diff
@@ -222,6 +223,8 @@ gint main_runShadow(gint argc, gchar* argv[]) {
         }
     }
 
+    disable_aslr();
```

Comment:
A comment here might be useful to state that we disable ASLR for determinism reasons, and not because we need to (for the memory manager for example).

Reply:
+1

Reply:
fixed!